### PR TITLE
2998 set default starting geolocation coordinates

### LINKED ISSUE #2998: Set default starting geolocation coordinates
**Use case**: I would like to be able to choose a default starting place on the map in entities with geolocation coordinates. Current default is Switzerland.

### PR DESCRIPTION
**Pull request by mfacar**
fixes #2998

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/react/Settings/components/CollectionSettings.js [hunk 1]
```diff
@@ -273,6 +273,12 @@ export class CollectionSettings extends Component {
               </label>
               <Control.text model=".mapTilerKey" className="form-control" />
             </div>
+            <div className="form-group">
+              <label className="form-group-label" htmlFor="defaultGeolocation">
+                {t('System', 'MAP STARTING POINT')}
```
[the remark this answers is not in the document]
Comment (mfacar) [reply app/react/Settings/components/CollectionSettings.js:278]: @konzz please can you extend about the use of map starting point component? I didn't get this :(

[the remark this answers is not in the document]
Comment (konzz) [reply app/react/Settings/components/CollectionSettings.js:278]: Updated the comment

File: app/api/settings/routes.js [hunk 2]
```diff
@@ -58,6 +58,15 @@ export default app => {
             favorites: Joi.boolean(),
           }),
           mapTilerKey: Joi.string().allow(''),
+          mapStartingPoint: Joi.array().items(
+            Joi.object().keys({
+              lat: Joi.number(),
+              lon: Joi.number(),
+              label: Joi.string()
```
[the remark this answers is not in the document]
Comment (mfacar) [reply app/api/settings/routes.js]: @konzz given there is a pending commit c3dc9a4 with a migration of this file to typescript that includes the validation to an AJV schema, I think is could be better waiting for the merge in order to include the new property. What do you think?

File: app/react/Settings/components/CollectionSettings.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mfacar) [reply app/react/Settings/components/CollectionSettings.js]: @konzz I've changed the translations of the component in order to use the Translate one. Is it necessary to include a migration to the database with the used labels?

[the remark this answers is not in the document]
Comment (konzz) [reply app/react/Settings/components/CollectionSettings.js]: No, it's not

**Comment by mfacar** [comment]
@konzz, I've returned this pr to draft after @fnocetti helped me to detect that the current map component is breaking the suite1 of e2e tests. Map Tiler Api is returning an error and this is preventing the execution of the rest of the tests.

File: app/react/Settings/components/CollectionSettings.js [hunk 3]
```diff
@@ -269,11 +283,27 @@ export class CollectionSettings extends Component {
...
+              </div>
+            </div>
+            <div className="form-group">
+              <label className="form-group-label" htmlFor="mapStartingPoint">
+                <Translate>Map starting point</Translate>
+              </label>
+              <Geolocation model=".mapStartingPoint" />
+              <div className="alert alert-info">
+                <div className="force-ltr">
+                  You can set the default starting point for your geolocation properties.
```
Comment (mfacar) [inline app/react/Settings/components/CollectionSettings.js:300]: @konzz I've added an info message about the options for the maps, please can you take a look about the correctness of it.
As #3066 is asking about to link settings with documentation, maybe it's required to update this link with these new options [Managing Settings](https://uwazi.readthedocs.io/en/initial-setup/admin-docs/managing-settings.html?highlight=PRIVATE%20INSTANCE#how-to-edit-your-site-settings)

**Pull request merged**